sdk/core/core-https/src/nodeHttpsClient.ts

```diff
@@ -39,6 +39,8 @@ function isArrayBuffer(body: any): body is ArrayBuffer | ArrayBufferView {
 class ReportTransform extends Transform {
   private loadedBytes = 0;
   private progressCallback: (progress: TransferProgressEvent) => void;
+
+  // eslint-disable-next-line @typescript-eslint/ban-types
   _transform(chunk: string | Buffer, _encoding: string, callback: Function): void {
```

Comment:
Would that work instead of disabling the rule?
```suggestion
  _transform(chunk: string | Buffer, _encoding: string, callback: (error?: any) => void): void {
```

Reply:
Here we are overriding a method from the parent class which we do not own. So, thought it best to not change the signature compared to the same method in the parent class

Reply:
`Function`  may have came from the definition of _transform in  base.d.ts.

Would `(error?: Error) => void` be better?  The doc says it must be an `Error` object if an error occurred.